workflows/execute_workflows.sh

```diff
@@ -5,6 +5,8 @@ WORKFLOW_DIR="$ROOT"/workflows
 OCRD_WORKFLOW_DIR="$WORKFLOW_DIR"/ocrd_workflows
 WORKSPACE_DIR="$WORKFLOW_DIR"/workspaces
 
+set -eou pipefail
```

Comment:
```suggestion
set -euo pipefail
```
(because `pipefail` is an argument to `-o`.)